### ISSUE TITLE
Config file template

### ISSUE THREAD

**Issue by aodhneine** [tsoding/kgbotka#87]
The only way to know how to properly create `config.json` is to look at `src/KGBotka/Config.hs` which is really unintuitive.

It would be nice to have some kind of template or specification for this file.

**Comment by rexim** [comment]
@aodhneine yeah, I agree. I plan to update the config format in the future (specifically after adding the Discord support #12 ). I'll document it after it stabilizes a little bit. Thanks!
